src/frontend/packages/lib_video/src/components/common/DashboardControlPane/DashboardTabStatistics/DashboardTabStatistics.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, Grid, Text } from 'grommet';
+import { useCurrentVideo } from 'hooks';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { useStatsVideo } from 'hooks/useVideoStats';
+
+const messages = defineMessages({
+  nbViewer: {
```

Comment:
Maybe a better message key would be something like `viewersStatsDescription` ? WDYT ?